### PR TITLE
RC 0.3.5

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

RC for v0.3.5

# Checklist:

- [x] Update the version number
- [x] Verify and update Zenodo information in .zenodo.json
- [x] doc tests pass locally
- [x] activate this branch on readthedocs (include link [here](https://pysatmissions.readthedocs.io/en/rc_0_3_5/))
- [x] Upload the test distribution to the [testPyPi server](https://test.pypi.org/project/pysatMissions/0.3.5/)
- [x] Contacted developers and users about the future release
- [x] Performed integration tests with downstream packages (include list of these packages)
- [x] Create a wiki page with the release text (put link [here](https://github.com/pysat/pysatMissions/wiki/v0.3.5))
- [x] After approval, confirm all changes from the RC pull are listed in the wiki release text.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes
- [x] Update zenodo.json file for new code contributors

If this is a release PR, replace the first item of the above checklist with the
release checklist on the pysat wiki:
https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

**Review by aburrell — APPROVED** [review]

**Review by rstoneback — COMMENTED** [review]
Looks good! Sorry for the delay. Saw this review request in my messages today.

One request, can my institution in the zenodo document be updated to Cosmic Studio ?

**Comment by jklenzing** [comment]
> Looks good! Sorry for the delay. Saw this review request in my messages today.
> 
> One request, can my institution in the zenodo document be updated to Cosmic Studio ?

Thanks for catching that. Fixed it.

**Review by rstoneback — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
+## [0.3.5] - 2024-05-10
```
Comment (jklenzing) [inline CHANGELOG.md]:
```suggestion
## [0.3.5] - 2024-07-12
```

**Comment by jklenzing** [comment]
@aburrell, I was updating the date to finish this off and found a new apexpy related bug:
```
E           NameError: name 'fa' is not defined
```

**Comment by aburrell** [comment]
Looks like that could be related to the release of numpy 2.0.0 😿.  It'll probably (?) be fixed if you limit the numpy version to 1.X.

On a more positive note, I think if you specify the compilers, you can get apexpy installed on the MacOS tests.

**Comment by aburrell** [comment]
Ok, I fixed the MacOS installation, but couldn't get GA to downgrade numpy for the apexpy install.  So, I toggled current testing to NEP29 only. This lead to the uncovering of a bug, where you were never testing NEP29.  So I also fixed NEP29.

The apexpy numpy 2.0.0 issues are documented, and it can run with numpy 2.0.0 if you install with a lower version of numpy.  So we'll work on that for a new release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
+## [0.3.5] - 2024-07-12
```
Comment (jklenzing) [inline CHANGELOG.md]:
```suggestion
## [0.3.5] - 2024-07-16
```

**Pull request merged**